cabal.project

```diff
@@ -0,0 +1,5 @@
+packages:   streamly.cabal
+          , benchmark/streamly-benchmarks.cabal 
+benchmarks: True
```

Comment:
Why do we need this? Will this cause benchmarks to be built all the time? If so, that's not what we want. We can always use `--enable-benchmarks` CLI option.

Reply:
Yeah, we don't need ``benchmarks: True`` I'll remove it. I'll add ``cabal.project`` to ``extra-source-files``